src/assets/stylesheets/extensions/_codehilite.scss

```diff
@@ -216,15 +216,23 @@ $codehilite-whitespace: transparent;
   // If code blocks are wrapped with codehilite, the styles must be adjusted
   // so the marker stretches to the whole width and the padding is respected
   .codehilite {
+    position: relative;
     margin: 1em 0;
-    padding: 1rem 1.2rem 0.8rem;
+    padding: 0;
     border-radius: 0.2rem;
     background-color: $md-code-background;
     color: $md-code-color;
     line-height: 1.4;
-    overflow: auto;
```

Comment:
I need to validate, that everything works as expected here. `overflow` is a critical property for Material, as it induces margin collapse which may break other features.